mobius_function.py

```diff
@@ -0,0 +1,43 @@
+"""

```

Comment:
You have two files: __mobius_function.py__ and __maths/mobius_function.py__.  This file needs to be removed